app/controllers/follow_requests_controller.rb

```diff
@@ -1,11 +1,10 @@
 class FollowRequestsController < ApplicationController
   before_action :set_follow_request, only: %i[ show edit update destroy ]
-  before_action :authorize_follow_request, only: %i[accept destroy]
+  before_action -> { authorize @follow_request || FollowRequest }, only: [:show, :edit, :update, :destroy]
```

Comment:
Great job refactoring your controller to make your code more concise! Please double check your restrictions as it seems like a current user isn't authorized to accept pending requests.